Document/0x06d-Testing-Data-Storage.md

```diff
@@ -183,9 +183,21 @@ The [`NSUserDefaults`](https://developer.apple.com/documentation/foundation/nsus
 - `NSSearchPathForDirectoriesInDomains, NSTemporaryDirectory`: used to manage file paths
 - `NSFileManager`: lets you examine and change the contents of the file system. You can use `createFileAtPath` to create a file and write to it.
 
-The following example shows how to create a securely encrypted file using the `createFileAtPath` method:
+The following example shows how to create a securely encrypted file using the `FileManager` class.
 
-```objectivec
+Swift:
+
+```swift
+FileManager.default.createFile(
+    atPath: filePath,
+    contents: Data(base64Encoded: "secret text"),
```

Comment:
let's please remove base64Encoded, this is otherwise misleading when we are talking earlier about a "secure" way of storing data and encrypting it.